plugins/woocommerce/client/admin/client/task-lists/fills/Marketing/index.tsx

```diff
@@ -164,6 +165,20 @@ const Marketing: React.FC< MarketingProps > = ( { onComplete } ) => {
 		actionTask( 'marketing' );
 	};
 
+	useEffect( () => {
+		if ( window.wcTracks.isEnabled ) {
+			recordEvent( 'task_marketing_marketplace_promo_shown', {
+				task: 'marketing',
+			} );
+		}
+	}, [] );
```

Comment:
Just an idea: maybe instead of checking if `window.wcTracks.isEnabled` is true, we can move the `recordEvent` call into the `TaskPromo` component. So when it gets rendered, we fire the Tracks event.